test/integration/features/addon-registry-route.feature

```diff
@@ -0,0 +1,50 @@
+@addon-registry-route @addon
+Feature: registry-route add-on
+registry-route addon exposes the route for the OpenShift integrated registry,
+so that user can able to push their container images directly to registry and deploy it.
+
+  Scenario: As a user, I can enable the registry-route add-on
+    Given user starts shell instance on host machine
```

Comment:
Yes we need this because ```docker-env``` and ```oc-env``` need to be configured in the same shell instance otherwise command like ```docker pull, docker tag``` which is being used in next [scenario](https://github.com/minishift/minishift/pull/2158/files#diff-1961420bfd9b54a7d22966ffc66161c6R23) will fail.